common/templates/student/edx_ace/accountactivation/email/body.html

```diff
@@ -17,7 +17,7 @@
             </p>
         {% endif %}
 
-        <p class="heading" style="font-size: 23pt;">{% blocktrans %}Hello {{ name }}{% endblocktrans %}</p>
+        <p class="heading" style="font-size: 23pt;">{{ username }} {% blocktrans %}Hello{% endblocktrans %}</p>
```

Comment:
why isn't the username inside the `blocktrans` like `{{ name }}`

Reply:
it causes RTL issue